amplify/backend/api/petrakipApi4data/schema.graphql

```diff
@@ -0,0 +1,95 @@
+# General Types and Enums
+
+type S3Object {
+  bucket: String!
+  key: String!
+  region: String!
+}
+
+enum ContentType {
+  image
+  video
+  text
+  audio
+}
+
+type Comment {
+  createdAt: AWSDateTime!
+  content: String!
+}
+
+type OrientationQuestions {
+  question: String!
+}
+
+# Model types -> create DynamoDB tables
+
+type ProfileSettings @model @auth(rules: [{ allow: owner }]) {
+  id: ID!
+  profileImage: S3Object
+  age: Int
+}
```

Comment:
@manzanillo it would be easier to discuss if you comment on this line. 

> This is missing the institution. Age is not needed.

the institution is saved inside Cognito -> these are the additional ones. Therefore I added age just to have something there in the beginning.